lib/dry/web/roda/templates/monitor.rb.tt

```diff
@@ -1,3 +1,6 @@
+require 'dry-monitor'
+Dry::Monitor.load_extensions(:sql)
```

Comment:
Maybe more logical will be to put this on boot/persistent where we boot rom-sql?

Reply:
Yes! This makes more sense.

Reply:
Actually I could also put it in the `init` block, where it's required below here